core/sql/arkcmp/CmpStoredProc.cpp

```diff
@@ -363,7 +361,7 @@ NABoolean CmpSPOutputFormat::getColumnDesc(SP_FIELDDESC_STRUCT* fDesc,
     {
       // convert ElemDDLColViewDef into CatColumn, then column_desc
       if ( !ElemDDLColDef2ColumnDescStruct
-	   (elemDDL, tableDesc_->body.table_desc.tablename, colsDesc) )
+	   (elemDDL, tableDesc_->tableDesc()->tablename, colsDesc) )
```

Comment:
For this checkin, wanted to keep the code structure somewhat similar to how it was.
That made it easier to make changes (lots of search and replace).
The old code used desc_struct which was replaced by TrafDesc class in the new code.
For a later checkin, we can change to using the underlying class instead of base class.